packages/xo-server/src/api/host.mjs

```diff
@@ -3,6 +3,7 @@ import { createLogger } from '@xen-orchestra/log'
 import assert from 'assert'
 import { format } from 'json-rpc-peer'
 import { incorrectState } from 'xo-common/api-errors.js'
+import { IPMI_CACHE } from '@xen-orchestra/xapi/host/_ipmi.mjs'
```

Comment:
The cache must be declared in `xo-server`'s code.

The `host/_ipmi.mjs` is a private mode that should not be imported directly.